modules/openapi-generator/src/main/resources/jmeter-client/api.mustache

```diff
@@ -110,7 +110,7 @@
               </elementProp>{{/queryParams}}{{#bodyParam}}
               <elementProp name="{{paramName}}" elementType="HTTPArgument">
                 <boolProp name="HTTPArgument.always_encode">false</boolProp>
-                {{=<% %>=}}<stringProp name="Argument.value">${<% paramName %>}</stringProp><%={{ }}=%>
+                {{=<% %>=}}<stringProp name="Argument.value">${__javaScript("${<% paramName %>}".replace(/'/g\, '"'),)}</stringProp><%={{ }}=%>
```

Comment:
@kannkyo thanks for the fix. I think one limitation with such approach is that if the json data contains `'`, e.g. 
```json
{
  "name": "John O'Shea"
}
```

the result will not work as it will become:

```json
{
  "name": "John O"Shea"
}
```

I'm happy to merge this fix while looking for better approach to address cases like the above.